tools/install-unprivileged.sh

```diff
@@ -330,7 +330,7 @@ if [ "$DIST" = el7 ]; then
 		EPELUTILS="$EPELUTILS fuse2fs"
 	fi
 elif [ "$DIST" = el8 ]; then
-	OSUTILS="$OSUTILS lzo squashfs-tools libzstd fuse3-libs"
+	OSUTILS="$OSUTILS lzo squashfs-tools libzstd fuse3-libs shadow-utils-subid"
```

Comment:
It looks like shadow-utils-subid also requires one more non-standard package, semanage (here and for el9+ below)

Reply:
Added semanage.

Reply:
Oops, that should be libsemanage.

Reply:
Next it's wanting the libxcrypt package

Reply:
Ok think I got all the dependent packages. Tested this locally and it seems to be working now.

Reply:
There's a subtle remaining problem.  The libxcrypt package happened to have its library in /lib64 instead of /usr/lib64.  Make this additional change:
```
diff --git a/tools/install-unprivileged.sh b/tools/install-unprivileged.sh
index a69840764..bdbcd1ff3 100755
--- a/tools/install-unprivileged.sh
+++ b/tools/install-unprivileged.sh
@@ -421,6 +421,7 @@ rmdir lib 2>/dev/null || true
 # move everything needed out of tmp to utils
 mkdir -p utils/bin utils/lib utils/libexec
 mv tmp/usr/lib*/* utils/lib
+mv tmp/lib*/* utils/lib 2>/dev/null || true # optional
 mv tmp/usr/*bin/*squashfs utils/libexec
 mv tmp/usr/*bin/fuse* utils/libexec 2>/dev/null || true # optional
 mv tmp/usr/bin/fake*sysv utils/bin
```